samples/word/01-basics/basic-api-call.yaml

```diff
@@ -13,13 +12,10 @@ script:
         async function run() {
             try {
                 await Word.run(async (context) => {
-                    const range = context.document.getSelection();
+                    let range = context.document.getSelection();
```

Comment:
FWIW, esp. on typescriptified (async/await) samples, I would definitely use "const" everywhere I can, "let" everywhere else, and "var" never.  So when you have the time, it'd be great if you can make those changes in the other samples as well.